FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,59 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			// to prevent memory leak
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data,response))
+
+			default:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+	
+	private final class FeedImagesMapper {
+		private struct Root: Decodable {
+			var items: [Image]
```

Comment:
`items` is never changed thus it should be `let`
```suggestion
			let items: [Image]
```